readme.md

```diff
@@ -174,6 +174,44 @@ Type: `object | Array | number | string | boolean | null` *(JSON-serializable va
 
 JSON body. If the `Content-Type` header is not set, it will be set to `application/json`.
 
+###### context
+
+Type: `unknown`
+
+User data. In contrast to other options, `context` is not enumerable.
```

Comment:
I think it needs some more non-code explanation than just "User data.". What is its purpose? Use-cases.

We should also mention that it's just passed through. Never merged. And Got never touches it.